lib/mavenlink/model.rb

```diff
@@ -223,6 +223,12 @@ def association_load_filters
       {}
     end
 
+    def subject
+      association_name = subject_type.pluralize
```

Comment:
`subject_type.pluralize` will not be a reliable way to build the GET URL here because different associations can return differently formatted subject types. For example, a custom field value `subject_type` is "workspace_group" and an external reference `subject_type` is "WorkspaceGroup".